src/objects/control.ts

```diff
@@ -84,21 +84,85 @@ export default class Control {
     return new Control(unflatten(flattened));
   }
 
-  toRuby() {
+  // WIP - provides the ability to get the control in its raw form
+  toString() {
```

Comment:
Is `.toString` being used?

Reply:
not in any call, but I used it for debug.